pkg/core/console/routes.go

```diff
@@ -37,13 +37,15 @@ func (c *Console) registerRoutes(logger *common.Logger, e *echo.Echo) {
 	g.GET("/node/:node", c.nodePage)
 	g.GET("/content", c.contentFragment)
 	g.GET("/content/upload", c.uploadPage)
+	g.GET("/uptime/:rollup/:validator", c.uptimeFragment)
 	g.GET("/uptime/:rollup", c.uptimeFragment)
 	g.GET("/uptime", c.uptimeFragment)
 	g.GET("/pos", c.posFragment)
 	g.GET("/block/:block", c.blockPage)
 	g.GET("/tx/:tx", c.txPage)
 	g.GET("/genesis", c.genesisPage)
 	g.GET("/health_check", c.getHealth)
+	g.GET("/adjudicate/:sp", c.adjudicateFragment)
```

Comment:
if an sp has multiple endpoints, what happens here? this is keyed by owner wallet right?